app/Actions/User/SaveUserInfoAction.php

```diff
@@ -99,8 +100,8 @@ private function storeAvatar(int $userId, ?UploadedFile $avatar): string
     {
         $avatarUrl = "";
         if ($avatar !== null) {
-            $avatarUrl = $userId . '.' . $avatar->extension();
-            $avatar->storeAs('public/upload/avatar', $avatarUrl);
+            Storage::putFileAs('public/upload/avatar', $avatar, $userId . '.' . $avatar->extension());
```

Comment:
Use FileNameGenerator class for fileName. This could be `public/upload/avatar` const value in FileNameGenerator.
File visibility param is required `Storage::putFileAs('upload/review', $file, $newFileName, 'public');` .